include/swift/IDE/Utils.h

```diff
@@ -311,12 +311,12 @@ enum class LabelRangeType {
 enum class ResolvedLocContext { Default, Selector, Comment, StringLiteral };
 
 struct ResolvedLoc {
-  CharSourceRange Range;
-  std::vector<CharSourceRange> LabelRanges;
-  llvm::Optional<unsigned> FirstTrailingLabel;
-  LabelRangeType LabelType;
-  bool IsActive;
-  ResolvedLocContext Context;
+  CharSourceRange range;
+  std::vector<CharSourceRange> labelRanges;
+  llvm::Optional<unsigned> firstTrailingLabel;
+  LabelRangeType labelType;
+  bool isActive;
+  ResolvedLocContext context;
```

Comment:
Could always use `SWIFT_NAME` to expose lowercase versions to Swift. That being said, given how inconsistent we are about casing in C++, I don't know if it's worth it.

Reply:
My impression is that we are transitioning to lowercase variable names anyway, so this was a good excuse.